kubeshop/testkube-dashboard
# Use GTM to streamline telemetry

## Changes

-

## Fixes

- https://github.com/kubeshop/testkube/issues/4118

## How to test it

- Check how it works with GTM disabled in [**Vercel preview**](https://testkube-dashboard-git-dawid-featuse-gtm-only-testkube-cloud.vercel.app/tests?~disable_telemetry=false&~debug_telemetry=true&~google_analytics_id=) - you may check console for telemetry debug; you need to use cluster with telemetry enabled (i.e. own local), because dev.testkube.io has telemetry disabled
- Check how it works with GTM enabled in [**Vercel preview**](https://testkube-dashboard-git-dawid-featuse-gtm-only-testkube-cloud.vercel.app/tests?~disable_telemetry=false&~debug_telemetry=true&~api_server_endpoint=https://dev.testkube.io/results/v1&~google_analytics_id=GTM-5QNRGGF) - you may check console for telemetry debug
- Don't forget to disable AdBlock

## screenshots

-

## Checklist

- [x] updated the docs - [**HERE**](https://github.com/kubeshop/testkube-dashboard/blob/dawid/feat/use-gtm-only/telemetry.md)
